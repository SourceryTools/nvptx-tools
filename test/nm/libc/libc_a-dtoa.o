// BEGIN PREAMBLE
.version 3.1
.target sm_35
.address_size 64
// END PREAMBLE
// BEGIN FUNCTION DECL: quorem
.func (.param .u32 %value_out) quorem (.param .u64 %in_ar0, .param .u64 %in_ar1);
// BEGIN GLOBAL FUNCTION DECL: _dtoa_r
.visible .func (.param .u64 %value_out) _dtoa_r (.param .u64 %in_ar0, .param .f64 %in_ar1, .param .u32 %in_ar2, .param .u32 %in_ar3, .param .u64 %in_ar4, .param .u64 %in_ar5, .param .u64 %in_ar6);
// BEGIN GLOBAL VAR DECL: __mprec_bigtens
.extern .const .align 8 .u64 __mprec_bigtens[];
[...]
// BEGIN VAR DEF: $LC2
.const .align 1 .u8 $LC2[2] =
{48,0 };
// BEGIN FUNCTION DEF: quorem
.func (.param .u32 %value_out) quorem (.param .u64 %in_ar0, .param .u64 %in_ar1)
{
[...]
}
// BEGIN GLOBAL FUNCTION DEF: _dtoa_r
.visible .func (.param .u64 %value_out) _dtoa_r (.param .u64 %in_ar0, .param .f64 %in_ar1, .param .u32 %in_ar2, .param .u32 %in_ar3, .param .u64 %in_ar4, .param .u64 %in_ar5, .param .u64 %in_ar6)
{
[...]
}
