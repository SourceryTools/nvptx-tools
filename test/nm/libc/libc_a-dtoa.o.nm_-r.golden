0000000000000000 t quorem
0000000000000000 T _dtoa_r
                 U __mprec_bigtens
0000000000000000 d $LC2
